frame/assets/src/tests.rs

```diff
@@ -556,3 +556,76 @@ fn imbalances_should_work() {
 		assert_eq!(Assets::total_supply(0), 30);
 	});
 }
+#[test]
+fn force_metadata_should_work() {
+	new_test_ext().execute_with(|| {
+		//force set metadata works
+		assert_ok!(Assets::force_create(Origin::root(), 0, 1, true, 1));
+		assert_ok!(Assets::force_set_metadata(Origin::root(), 0, vec![0u8; 10], vec![0u8; 10], 8, false));
+		assert!(Metadata::<Test>::contains_key(0));
+
+		//overwrites existing metadata
+		let asset_original_metadata = Metadata::<Test>::get(0);
+		assert_ok!(Assets::force_set_metadata(Origin::root(), 0, vec![1u8; 10], vec![1u8; 10], 8, false));
+		assert_ne!(Metadata::<Test>::get(0), asset_original_metadata);
+
+		//attempt to set metadata for non-existent asset class
+		assert_noop!(
+			Assets::force_set_metadata(Origin::root(), 1, vec![0u8; 10], vec![0u8; 10], 8, false),
+			Error::<Test>::Unknown
+		);
+
+		//string length limit check
+		let limit = StringLimit::get() as usize;
+		assert_noop!(
+			Assets::force_set_metadata(Origin::root(), 0, vec![0u8; limit + 1], vec![0u8; 10], 8, false),
+			Error::<Test>::BadMetadata
+		);
+		assert_noop!(
+			Assets::force_set_metadata(Origin::root(), 0, vec![0u8; 10], vec![0u8; limit + 1], 8, false),
+			Error::<Test>::BadMetadata
+		);
+
+		//force clear metadata works
+		assert!(Metadata::<Test>::contains_key(0));
+		assert_ok!(Assets::force_clear_metadata(Origin::root(), 0));
+		assert!(!Metadata::<Test>::contains_key(0));
+
+		//Error handles clearing non-existent asset class
+		assert_noop!(Assets::force_clear_metadata(Origin::root(), 1), Error::<Test>::Unknown);
+		
+	});
+}
+
+#[test]
+fn force_asset_status_should_work(){
+	new_test_ext().execute_with(|| {
+		Balances::make_free_balance_be(&1, 10);
+		assert_ok!(Assets::create(Origin::signed(1), 0, 1, 30));
+		assert_ok!(Assets::mint(Origin::signed(1), 0, 1, 100));
+
+		//force asset status to change min_balance > balance 
+		assert_ok!(Assets::force_asset_status(Origin::root(), 0, 1, 1, 1, 1, 101, true, false));
+		assert_eq!(Assets::balance(0, 1), 100);
+		//asset in practice becomes frozen until balance >= min_balance 
```

Comment:
Allows for accounts to have less than min balance, not sure if that is problematic or not

Reply:
What happens when you transfer 1 asset to this user. Will they accept the 1 asset or will it kill the account?